todo/index.js

```diff
@@ -0,0 +1,5 @@
+// dom
+const $todayInput = document.querySelector(".today-input");
+const $tomorrowInput = document.querySelector(".tomorrow-input");
+const $todayButton = document.querySelector(".today-button");
+const $tomorrowButton = document.querySelector(".tomorrow-button");
```

Comment:
```suggestion
const $ = (selector) => document.querySelector(selector);

const todayInput = $(".today-input");
const tomorrowInput = $(".tomorrow-input");
const todayButton =$(".today-button");
const tomorrowButton = $(".tomorrow-button");
```
이런식으로도 편하게 변수명 쓸수 이따!!

Reply:
와 이건 처음 알았다. 너무 고마워욧 메모